sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1208,6 +1211,231 @@ case class GpuToUTCTimestamp(
   }
 }
 
+class MonthsBetweenExprMeta(
+                              expr: MonthsBetween,
+                              override val conf: RapidsConf,
+                              override val parent: Option[RapidsMeta[_, _, _]],
+                              rule: DataFromReplacementRule)
+  extends ExprMeta[MonthsBetween](expr, conf, parent, rule) {
+
+  override def isTimeZoneSupported = true
+
+  override def convertToGpu(): GpuExpression = {
+    val gpuChildren = childExprs.map(_.convertToGpu())
+    assert(gpuChildren.length == 3)
+    GpuMonthsBetween(gpuChildren(0), gpuChildren(1), gpuChildren(2), expr.timeZoneId)
+  }
+}
+
+object GpuMonthsBetween {
+  val UTC = ZoneId.of("Z")
+
+  private def convertToZoneAndClose(micros: GpuColumnVector, zoneId: ZoneId): ColumnVector = {
+    withResource(micros) { _ =>
+      if (zoneId.equals(UTC)) {
```

Comment:
```suggestion
      if (zoneId.normalized.equals(UTC)) {
```
Note that there's GpuOverrides.isUTCTimezone helper methods for this kind of checking.

Reply:
I already normalized it in a common place on line 1393.